packages/http-client/src/csrf-token-header.spec.ts

```diff
@@ -172,6 +172,31 @@ describe('buildCsrfHeaders', () => {
     });
     expect(headers).toEqual(expected);
   });
+
+  it('should try csrf request with destination url when request config url is not set', async () => {
+    const request = await createRequestBuilder({
+      payload: new CommonEntity(commonEntityApi)
+    })['build'](defaultDestination);
+    const mockedHeaders = {
+      'x-csrf-token': 'mocked-x-csrf-token',
+      'set-cookie': ['mocked-cookie-0;mocked-cookie-1', 'mocked-cookie-2']
+    };
+
+    nock(defaultHost)
+      .head(request.serviceUrl() + '/')
+      .reply(200, undefined, mockedHeaders);
+
+    const expected = {
+      cookie: 'mocked-cookie-0;mocked-cookie-2',
```

Comment:
[q] why is the `mocked-cookie-1` not set?

Reply:
The logic [here](https://github.com/SAP/cloud-sdk-js/blob/c370baa7b7c919969b895be13b1f822db4991569/packages/http-client/src/csrf-token-header.ts#L158) only picks the first string from each item in the array.